### PR TITLE
LabWork2_AleksandraMihaylova

### PR DESCRIPTION
**Pull request by aleksandramihaylova**

### CODE REVIEW

File: LabWork2_AleksandraMihaylova /LabWork2_AleksandraMihaylova.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+where continent = 'Asia';
+
+select *
+from country
+where Continent in ('Asia', 'Africa') and Population >= 100000000;
+
+select *
+from country
+where Name in ('Andorra', 'Barbados', 'Yemen', 'Vietnam', 'Poland');
+
```
Comment (sslavova) [inline LabWork2_AleksandraMihaylova /LabWork2_AleksandraMihaylova.sql:14]: Just a small fix - keep in mind you need all kinds of republics :)

**Review by sslavova — CHANGES_REQUESTED** [review]
Just one small update needed for the government form, everything else looks good :)

File: LabWork2_AleksandraMihaylova /LabWork2_AleksandraMihaylova.sql (hunk 1 above)
Comment (aleksandramihaylova) [reply LabWork2_AleksandraMihaylova /LabWork2_AleksandraMihaylova.sql:14]: I edited the file!

**Review by sslavova — APPROVED** [review]
Looks good now :)

**Pull request merged**